apps/src/applab/setPropertyDropdown.js

```diff
@@ -53,10 +53,49 @@ var PROP_INFO = {
   step: { friendlyName: 'step', internalName: 'step', type: 'number' }
 };
 
+let DEFAULT_PROP_VALUES = {
+  width: '100',
+  height: '100',
+  x: '100',
+  y: '100',
+  'text-color': '"red"',
+  'background-color': '"red"',
+  'font-size': '100',
+  'text-align': '"left"',
+  hidden: 'true',
+  text: '"text"',
+  placeholder: '"text"',
+  image: '"https://code.org/images/logo.png"',
+  'icon-color': '"red"',
+  'group-id': '"text"',
+  checked: 'true',
+  readonly: 'true',
+  options: '["option1", "etc"]',
+  value: '100',
+  min: '100',
+  max: '100',
+  step: '100',
+};
+
 // When we don't know the element type, we display all possible friendly names
-var fullDropdownOptions = _.uniq(Object.keys(PROP_INFO).map(function (key) {
-  return '"' + PROP_INFO[key].friendlyName + '"';
-}));
+var fullDropdownOptions = _.uniqBy(Object.keys(PROP_INFO).map(function (key) {
```

Comment:
Nit: I had a hard time parsing this block as formatted, especially picking out where the chained operations of the first argument ended and the second argument began.  I'm not sure what would be better though.  Maybe this?
```js
var fullDropdownOptions = _.uniqBy(
  Object
    .keys(PROP_INFO)
    .map((key) => {
      // ...contents of map function...
    })
    .filter(object => object),

  // Uniqueness key iteratee:
  object => object.text
);
```
Also 100% possible I'm overthinking this.